package.json

```diff
@@ -9,20 +9,22 @@
     "version": "independent"
   },
   "scripts": {
+    "dev": "vite",
+    "serve": "vite preview",
+    "preview:prod": "MODE=production yarn build:web && vite preview",
+    "preview:dev": "MODE=development yarn build:web && vite preview",
     "analyze": "ANALYZE=true yarn build:web",
     "bump": "yarn upgrade-interactive --latest --scope @shapeshiftoss",
-    "build:web": "yarn build:packages && NODE_OPTIONS=--max-old-space-size=8192 IMAGE_INLINE_SIZE_LIMIT=0 INLINE_RUNTIME_CHUNK=false DISABLE_ESLINT_PLUGIN=true react-app-rewired --openssl-legacy-provider build && tsx ./scripts/writeHeaders.ts && ./scripts/sha256sums.sh && ./scripts/verifyWebpackHashes.sh && tsx ./scripts/writeBuildMetadata.ts",
+    "build:web": "yarn build:packages && NODE_OPTIONS=--max-old-space-size=12288 IMAGE_INLINE_SIZE_LIMIT=0 INLINE_RUNTIME_CHUNK=false DISABLE_ESLINT_PLUGIN=true vite build --mode=${MODE:-production} && ./scripts/sha256sums.sh && ./scripts/verifyViteHashes.sh && tsx ./scripts/writeBuildMetadata.ts",
     "build:packages": "yarn clean:packages && yarn workspace @shapeshiftoss/unchained-client generate && yarn tsc --build tsconfig.packages.json",
     "clean": "yarn clean:packages && yarn clean:web && rm -rf node_modules coverage tsconfig.tsbuildinfo",
     "clean:web": "rm -rf build tsconfig.web.tsbuildinfo",
     "clean:packages": "yarn workspaces foreach -ptiv --exclude @shapeshiftoss/web run clean && rm -rf tsconfig.packages.tsbuildinfo",
     "codemod:clear-assets-migration": "jscodeshift -t src/state/migrations/transformClearAssets.ts --parser=ts --extensions=ts src/state/migrations/index.ts",
     "deadcode": "yarn ts-prune",
-    "dev": "yarn concurrently -n \"packages,web\" \"yarn dev:packages\" \"yarn dev:web\"",
```

Comment:
Note - I am not seeing hot reloading when making changes to packages

Reply:
Yep that's absolutely expected for the time being unfortunately, @NeOMakinG can comment on further steps we could take to make this work as discussed IRL!

Reply:
Nice catch, fixed in [780fc35](https://github.com/shapeshift/web/pull/8924/commits/780fc358dd698fa1d4f35579acc5f620dc5da98a)